cmd/juju/caas/add.go

```diff
@@ -135,12 +137,16 @@ type AddCAASCommand struct {
 	// resourceGroup is the resource group name for the cluster.
 	resourceGroup string
 
-	// hostCloudRegion is the cloud region that the nodes of cluster (k8s) are running in.
-	// The format is <cloudType/region>.
+	// hostCloudRegion stores user specified value of a region option.
```

Comment:
The original comment is still valid - the point of this var is to record where the cluster nodes are running. We try and query it but require the user to specify if it is unable to be determined automatically.